practice-one/src/index.html

```diff
@@ -9,6 +9,19 @@
   </head>
 <body class="container">
   <include src="./components/header.html"></include>
+  <include src="./components/food-card.html"
+  locals='{
+    "image": "./assets/images/kebab.png",
+    "imageAlt":"Kebab image",
+    "rating1": "./assets/images/rating-1.png",
+    "rating2": "./assets/images/rating-2.png",
+    "rating3": "./assets/images/rating-3.png",
+    "ratingStar": "./assets/images/rating-star.png",
+    "rating": "(4.5)",
```

Comment:
do we need to pass bracket every time, can we just pass `4.5` or `.5` only

Reply:
thanks i fixed it